lib/charms/mysql/v0/mysql.py

```diff
@@ -276,6 +282,8 @@ def configure_mysql_users(self):
         configure_users_commands = (
             f"CREATE USER '{self.server_config_user}'@'%' IDENTIFIED BY '{self.server_config_password}'",
             f"GRANT ALL ON *.* TO '{self.server_config_user}'@'%' WITH GRANT OPTION",
+            f"CREATE USER '{self.exporter_user}'@'%' IDENTIFIED BY '{self.exporter_password}' WITH MAX_USER_CONNECTIONS 3",
+            f"GRANT PROCESS, REPLICATION CLIENT, SELECT ON *.* TO '{self.exporter_user}'@'%'",
```

Comment:
A good starting point is [this](https://github.com/percona/percona-server-mysql-operator/blob/2684de79e4bbba6221a5c08a71c4799f622c4fd9/build/ps-entrypoint.sh#L318-L321) for the necessary `GRANTS`

Reply:
yes, the link is correct, but without BACKUP_ADMIN
```suggestion
            f"CREATE USER '{self.exporter_user}'@'%' IDENTIFIED BY '{self.exporter_password}' WITH MAX_USER_CONNECTIONS 3",
            f"GRANT SYSTEM_USER, SELECT, PROCESS, SUPER, REPLICATION CLIENT, RELOAD ON *.* TO '{self.exporter_user}'@'%'",
```